server/package.json

```
@@ -22,7 +22,8 @@
     "test:cov": "jest --coverage",
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
     "test:e2e": "jest --config ./apps/immich/test/jest-e2e.json",
-    "typeorm": "node --require ts-node/register ./node_modules/typeorm/cli.js"
+    "typeorm": "node --require ts-node/register ./node_modules/typeorm/cli.js",
+    "api:generate-typescript": "rm -rf ../web/src/lib/open-api; node_modules/.bin/openapi-generator-cli generate -g typescript-fetch -i ./immich-openapi-specs.json -o ../web/src/lib/open-api"
```

Might be a good idea to use `&&` between commands, so if the first one fails it wouldn't run the second one. Also can use `npx` to run a package binary:
```suggestion
    "api:generate-typescript": "rm -rf ../web/src/lib/open-api && npx openapi-generator-cli generate -g typescript-fetch -i ./immich-openapi-specs.json -o ../web/src/lib/open-api"
```